lost/src/test/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImplTest.java

```diff
@@ -252,7 +251,7 @@ public void setMockLocation_shouldInvokeListenerOnce() throws Exception {
         assertThat(listener.getMostRecentLocation()).isEqualTo(mockLocation);
     }
 
-    @Test @Ignore("Find a better way to test this without Thread.sleep()")
```

Comment:
Unfortunately no :(

However by upgrading Robolectric I was able to reduce the sleep value (see below) and ensure with greater certainty that things will run in the order we expect them to using controls like `ShadowLooper.runUiThreadTasks()`.